### ISSUE TITLE
Send controller and other trackers position to the server

### ISSUE THREAD

**Issue by Eirenliel** [SlimeVR/SlimeVR-OpenVR-Driver#1]
Need to send controller and other trackers position to the server to use them as refrence for other body parts (like elbows) or references instead of HMD.

**Comment by kitlith** [comment]
per https://github.com/ValveSoftware/openvr/issues/1473 it seems like the API provided to drivers does not give role information.

Would it make more sense to:
 - Make SlimeVR-Server an openvr application/(dashboard) overlay so that it can acquire the information itself?
 - Make an additional "feeder" openvr application/(dashboard) overlay that is separate from SlimeVR-Server?

Former makes sense to me from an integration standpoint (i.e. editing slimevr settings from your steamvr dashboard), latter might be easier to quickly prototype.

**Comment by Eirenliel** [comment]
Yeah vavle's arbitrary driver limitations are ridiculous. I don't have a plan right now to rework the whole slimevr to make it openvr app, both because this will be a hell to develop, and that SlimeVR is not planned as only open-vr.

There is another option: send data without role information, guestimate it by id, and let users rebind it if it's wrong.

**Comment by kitlith** [comment]
True that. Given that you don't want to make it openvr only, it makes sense to me to make a feeder application, and maybe there could be additional communication between feeder and server in the future for changing settings.

Is there anything that might break if sending hmd position back isn't precisely synced with the driver?

**Comment by Eirenliel** [comment]
@kitlith you don't need to send hmd back

**Comment by kitlith** [comment]
> @kitlith you don't need to send hmd back

sorry, i meant positions in general. i was kinda figuring in my head that we could send hmd in the same place as controllers and trackers, which would also be a good sanity check that <new tracking data feeding scheme> works well before needing to actually use the new controller/tracker data.

**Comment by kitlith** [comment]
I've got a start on a feeder application here: https://github.com/kitlith/SlimeVR-Feeder-App

still working on it of course, but i think I've covered enough of the input system so that we can actually start pushing/pulling actual pose data.

**Comment by TitansWhale** [comment]
after add tracker I found that we can‘t get the contoller data.
I did the following.
1. run the openvr [opengl_example](https://github.com/ValveSoftware/openvr/tree/master/samples/hellovr_opengl) (It could run that I can use controller to disable the background)
![image](https://user-images.githubusercontent.com/26743177/196620268-3d55fee7-63dd-4be0-8101-4e524ecb17c4.png)
2. close the opengle_example and open the slimevr-server 
![image](https://user-images.githubusercontent.com/26743177/196620531-e0921cfa-b25f-4f73-9611-d19486198b8f.png)
3. re run the opengl_example I found that the controller can't work.(I can't use controller to disable the background)
![image](https://user-images.githubusercontent.com/26743177/196621142-215785df-06b8-4b16-a50e-bd7eefb2b738.png)

I found that when we load the driver we can't get the controller data.          T-T

**Comment by Eirenliel** [comment]
Controller and other trackers data is sent via the feeder app: https://github.com/SlimeVR/SlimeVR-Feeder-App
This issue is no longer relevant, driver only sends HMD data.
